data_sources/perturb-seq/README.md

```diff
@@ -36,6 +36,15 @@ python3 ../elastic_load.py \
       "type": "keyword"
     }
   }'
+
+# Modify the maximum number of results that can be returned.
+curl -X PUT "${ELASTIC_ENDPOINT}/perturb-seq/_settings" -H 'Content-Type: application/json' -d'
+{
+  "index": {
+    "max_result_window": 1000000
+  }
+}
+'
```

Comment:
Please consider using search_after:
1.  From ES docs - 'We no longer recommend using the scroll API for deep pagination. If you need to preserve the index state while paging through more than 10,000 hits, use the [search_after](https://www.elastic.co/docs/reference/elasticsearch/rest-apis/paginate-search-results#search-after) parameter with a point in time (PIT).'
2. search_after docs - https://www.elastic.co/docs/reference/elasticsearch/rest-apis/paginate-search-results#search-after

Reply:
I've managed to get a good implementation of this, but it ended requiring a refactor significant enough that rebasing the entire stack of PRs would be a pain. So I'll merge this implementation as is, and the requested changes are submitted in this follow-up PR on the top of the stack: https://github.com/EMBL-EBI-ABC/PerturbationCatalogue/pull/229